ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -229,7 +229,15 @@ def update_policy(self):
             mean_return=float(np.mean(self.cumulative_returns_since_policy_update)),
         )
         self.cumulative_returns_since_policy_update = []
-        batch_size = self.trainer_parameters["batch_size"]
+
+        # Make sure batch_size is a multiple of sequence length
```

Comment:
Nit: can you expand the comment to explain why we need to make sure about this? E.g. it's a tensorflow requirement?

Reply:
Added